### ISSUE TITLE
Best Checkpoint - Reg

### ISSUE THREAD

**Issue by jyothsna-phd22** [open-mmlab/OpenLidarPerceptron#1702]
Hi,
I am working on point cloud based object detection using openpcdet. Based on the openpcdet code, after we run the model for the mentioned 80 or 100 epochs with learning rate decay, how to find best checkpoint that will give good test result compared to other trained epochs in official kitti server? Kindly help as the number of chances to submit test results to the server is limited to three per month.

**Comment by Petros626** [comment]
The submit to the official kitti metrics, the website give some hints. It's not a novelty to train a model here and submit new mAP or AP results. From what I understand you have to create something which brings a further novelty for 3d object detection combined with a paper.

Finding the best epoch would require evaluating metrics,maybe precision/recall or the model fitness

**Comment by jyothsna-phd22** [comment]
Thank You.
